src/vos/tests/vts_wal.c

```diff
@@ -898,14 +1061,13 @@ run_wal_tests(const char *cfg)
 	unsigned int	 val = 0;
 	int		 i, rc;
 
-
 	d_getenv_int("DAOS_MD_ON_SSD", &val);
 	if (val == 0) {
-		print_message("MD_ON_SSD isn't enabled, skip test\n");
-		wal_skip_tests = 1;
+		print_message("DAOS_MD_ON_SSD isn't enabled, skip all tests.\n");
+		return 0;
```

Comment:
If skipped, the count of the skipped tests would not be incremented. (vos_test prints the stats at the bottom.) The term 'skip' is confusing: the result may be not what's expected. Maybe say "omit WAL refill tests"?

Reply:
Agreed, the term 'skip' is kind of confusing, but intention here is to avoid increasing skipped number. I'll fix the message in follow-on patches.